src/AppBundle/Entity/User.php

```diff
@@ -258,8 +258,9 @@ class User implements UserInterface
      */
     public function __construct()
     {
-        $this->clients = new \Doctrine\Common\Collections\ArrayCollection();
+        $this->clients = new ArrayCollection();
         $this->password = '';
+        $this->teams = new ArrayCollection();
```

Comment:
my change was only initialising the teams, otherwise null and I couldn't safely call "isEmpty"